javascript/ql/src/experimental/Security/CWE-090/LdapInjection.qhelp

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>If an LDAP query is built using string concatenation or string formatting, and the
+components of the concatenation include user input without any proper sanitization, a user 
+is likely to be able to run malicious LDAP queries.</p>
+</overview>
+
+<recommendation>
+<p>If user input must be included in an LDAP query, it should be escaped to
+avoid a malicious user providing special characters that change the meaning
+of the query. In NodeJS, it is possible to build the LDAP query using frameworks like `ldapjs`. 
```

Comment:
Backticks don't work in qhelp files.
Use `<code>foo</code>` instead.
Similar instances below.

Reply:
Thanks. I made the change.